docs/maya/simple_setup.md

```diff
@@ -18,7 +18,7 @@ The AdnSkin deformer will get applied to the second mesh which will become the s
 
 ### Create Deformer
 
-To create the AdnSkin deformer select the target mesh and then the skin mesh. Then press the ![AdnSkins](images/adn_skin.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Skin*.
+To create the AdnSkin deformer select one or more target meshes and then the skin mesh. Then press the ![AdnSkins](images/adn_skin.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Skin*.
```

Comment:
I had a look at the code again and we only allow the creation of AdnSkin with 2 meshes selected. When we are allowed to remove and add targets. Reading the docs I was thinking if we should also allow to create AdnSkin without a target?

Reply:
Hey Lucas! 

(1) Most probably the branch did not get updated for you. I can confirm that we will be able now to create an AdnSkin deformer with more than one target mesh. Check lines 33. 53 and 71.

![image](https://github.com/user-attachments/assets/bd9ec62b-8caa-4eba-90f1-cbb49cf77405)

(2) About the second one, I thought about it, now that we allow removing all the targets and adding whenever we want we should be able to create the deformer without any target. 

The thing is that we allow to remove all the targets just for the case the user is not happy with the setup and wants to remove a lot of targets and remake it. On creation the user should not have this problem and should create the deformer with the minimum requirements.

Anyway, as know we are two that thought of this possibility, let me share it with @carlosmonteagudoinbibo.

Reply:
Sorry for the confusion Luis. I was referring to not being able to create the AdnSkin with just 1 node selected. I know that we can now create it with 2 or more selections.
About creating AdnSkin without target meshes. I thought of it because in theory we would be able to export the AAD file without any targets. Then when importing we kind of create an AdnSkin without a target mesh right? I'll leave this comment open for discussions. But in my opinion this is optional and we have to think of the deadlines

Reply:
I agree with modifying the apply_skin to allow creating AdnSkin selecting only the destination mesh. It should be a very small change and safe, right @luisalbaceteinbibo ?

CC @lucasferryinbibo @josevalverdeinbibo

Reply:
Yes, it is, and will be added before today's scrum

Reply:
Resolving this comment as we had already implemented it